### PR TITLE
Add inquiry checkout tap event

### PR DESCRIPTION
**Pull request by xtina-starr**
The type of this PR is: **feat**

This PR resolves [PURCHASE-2491]

### Description

This adds two helpers for the conversation inquiry checkout events: `tappedMakeOffer` and `tappedViewOffer`

### PR Checklist (tick all before merging)

- [x] If I've added a new file to the tree I've exported it from the common `index.ts`
- [x] I've added comments with examples for any new interfaces or helpers and ensured that they're in the docs 
- [x] No platform-specific terminology has been used outside of `click` and `tap` (platform is inferred by the DB storing events)

[PURCHASE-2491]: https://artsyproduct.atlassian.net/browse/PURCHASE-2491

### CODE REVIEW

**Review by iskounen — COMMENTED** [review]
Thanks for taking the initiative to bake-in event tracking early in the project 🎉

**Review by lilyfromseattle — APPROVED** [review]
Looks good! I like that you added the example comments. Is there a follow up ticket to hook these up?

**Comment by xtina-starr** [comment]
@lilyfromseattle maybe these could go in the sub-task tickets of our main analytics tickets.

File: src/Helpers/Conversations/TappedViewOffer.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * A user tapped the view offer CTA in an inquiry
+ *
+ * @example
+ * ```
+ * tappedViewOffer({ id: "123", cta: "Offer Accepted" })
+ * ```
+ *
+ */
+export const tappedViewOffer = ({
+  id,
```
Comment (starsirius) [inline src/Helpers/Conversations/TappedViewOffer.ts:19]: Non-blocking: `id` may not be clear about what object we are referring to. How about making it explicit with `impulseConversationId`?

**Review by starsirius — APPROVED** [review]
Had a non-blocking comment and looks great to me! Feel free to merge when ready!

**Comment by artsyit** [comment]
:rocket: PR was released in [`v1.77.0`](https://github.com/artsy/cohesion/releases/tag/v1.77.0) :rocket:

**Pull request merged**